src/sys/cleanup.c

```diff
@@ -134,6 +134,7 @@ static NTSTATUS FspFsvolCleanup(
         Request->Req.Cleanup.SetArchiveBit ||
         Request->Req.Cleanup.SetLastWriteTime ||
         Request->Req.Cleanup.SetChangeTime ||
+        FsvolDeviceExtension->VolumeParams.FlushOnCleanup ||
```

Comment:
This change is no longer necessary. The `VolumeParams.PostCleanupWhenModifiedOnly` field will be enough to get the FSD to always send the `Cleanup` request to user mode.